dozer/cogs/actionlogs.py

```diff
@@ -297,27 +320,136 @@ async def messagelogconfig(self, ctx, channel_mentions: discord.TextChannel):
+        else:
+            config[0] = CustomJoinLeaveMessages(guild_id=ctx.guild.id, ping=True)
+        await config[0].update_or_add()
+
+        e = discord.Embed(color=blurple)
+        e.add_field(name='Success!', value=f"Ping on join is set to: {config[0].ping}")
+        e.set_footer(text='Triggered by ' + ctx.author.display_name)
+        await ctx.send(embed=e)
+
+    @memberlogconfig.command()
+    @has_permissions(manage_guild=True)
+    async def setjoinmessage(self, ctx, *, template: str = None):
+        """Configure custom join message template"""
+        e = discord.Embed(color=blurple)
+        e.set_footer(text='Triggered by ' + ctx.author.display_name)
+        if template:
+            config = CustomJoinLeaveMessages(
+                guild_id=ctx.guild.id,
+                join_message=template
+            )
+            e.add_field(name='Success!', value=f"Join message template has been set to\n{template}")
+        else:
+            config = CustomJoinLeaveMessages(
+                guild_id=ctx.guild.id,
+                join_message=CustomJoinLeaveMessages.nullify
+            )
+            e.add_field(name='Success!', value="Join message has been set to default")
+        await config.update_or_add()
+        await ctx.send(embed=e)
+
+    @memberlogconfig.command()
+    @has_permissions(manage_guild=True)
+    async def setleavemessage(self, ctx, *, template=None):
+        """Configure custom leave message template"""
+        e = discord.Embed(color=blurple)
+        e.set_footer(text='Triggered by ' + ctx.author.display_name)
+        if template:
+            config = CustomJoinLeaveMessages(
+                guild_id=ctx.guild.id,
+                leave_message=template
+            )
+            e.add_field(name='Success!', value=f"Leave message template has been set to\n{template}")
+        else:
+            config = CustomJoinLeaveMessages(
+                guild_id=ctx.guild.id,
+                leave_message=CustomJoinLeaveMessages.nullify
+            )
+            e.add_field(name='Success!', value="Leave message has been set to default")
+        await config.update_or_add()
+        await ctx.send(embed=e)
+
+    @memberlogconfig.command()
+    @has_permissions(manage_guild=True)
+    async def disable(self, ctx):
+        """Disables Join/Leave logging"""
+        e = discord.Embed(color=blurple)
+        e.set_footer(text='Triggered by ' + ctx.author.display_name)
+        config = CustomJoinLeaveMessages(
+            guild_id=ctx.guild.id,
+            channel_id=CustomJoinLeaveMessages.nullify
+        )
+        await config.update_or_add()
+        e.add_field(name='Success!', value="Join/Leave logs have been disabled")
+        await ctx.send(embed=e)
+
+    @memberlogconfig.command()
+    @has_permissions(manage_guild=True)
+    async def help(self, ctx):  # I cannot put formatting example in example_usage because then it trys to format the example
```

Comment:
You can use double curly brackets to print literal curly brackets in `format` or f-strings. Read more https://docs.python.org/3/library/string.html#format-string-syntax